src/hooks/useFetch.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+const useFetch = (url, lat, lng) => {
+  const [data, setData] = useState([]);
+  const [status, setStatus] = useState('idle');
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (!url) return;
+
+    const getWikiEntries = async (lat, lng) => {
+      setStatus('loading');
```

Comment:
I really like the use of this string literal for the status. 'loading' 'idle', and 'error'. Very human readable.